microservices/cmslitemetadata_to_redshift/ddl/cmslite.metadata.sql

```diff
@@ -0,0 +1,200 @@
+    "value"   VARCHAR(255)     NOT NULL
+); */
+
+CREATE TABLE IF NOT EXISTS cmslite.audiences (
+    "id"      BIGINT		ENCODE LZO	NOT NULL,
+    "value"   VARCHAR(255)  ENCODE ZSTD	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- SY: Added this table to reflect cmslite_gdx.json
+CREATE TABLE IF NOT EXISTS cmslite.creators (
+    "id"      BIGINT		ENCODE LZO	NOT NULL,
+    "value"   VARCHAR(255)  ENCODE ZSTD	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- LOOKUP TABLES (MANY-TO-MANY)
+
+-- SY: dbtables_metadata
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_content_types (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_subject_categories (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_subjects (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_languages (
+     "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- SY: Table no longer in cmslite schema
+/* CREATE TABLE IF NOT EXISTS cmslite.metadata_security_classifications (
+    "node_id"   VARCHAR(255)    NOT NULL,
+    "id"        BIGINT          NOT NULL
+); */
+
+-- SY: Table no longer in cmslite schema
+/* CREATE TABLE IF NOT EXISTS cmslite.metadata_security_labels (
+    "node_id"   VARCHAR(255)    NOT NULL,
+    "id"        BIGINT          NOT NULL
+); */
+
+CREATE TABLE IF NOT EXISTS cmslite.metadata_audiences (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- SY: Added this table to reflect cmslite_gdx.json
+CREATE TABLE IF NOT EXISTS cmslite.metadata_creators (
+    "node_id"   VARCHAR(255)    ENCODE LZO		NOT NULL,
+    "id"        BIGINT          ENCODE DELTA	NOT NULL
+)  
+DISTSTYLE EVEN;
+
+-- METADATA TABLE FOR NODE_ID RECORDS
+-- NODE_ID should be primary key, however RedShift doesn't seem to enforce this.
+
+-- SY: Old table
+/* CREATE TABLE IF NOT EXISTS cmslite.metadata (
+    "node_id"           VARCHAR(255)     NOT NULL,
+    "keywords"          VARCHAR(1000)   ,
+    "description"       VARCHAR(1000)   ,
+    "page_type"         VARCHAR(255)    ,
+    "synonyms"          VARCHAR(1000)   ,
+    "dcterms_creator"   VARCHAR(255)    ,
+    "publication_date"  DATE            ,
+    "modified_date"     TIMESTAMP       ,
+    "created_date"      TIMESTAMP       ,
+    "updated_date"      TIMESTAMP       ,
+    "published_date"    TIMESTAMP       
+); */
+
+-- SY: New table
+-- SY: Not sure if node_id needs NOT NULL but kept it as it was in the old table above
+-- SY: sitekey in Looker but site_key in cmslite_gdx.json??
```

Comment:
Good catch. Not a breaking issue though, but I'm going to raise this in the ticket in Jira. Please remove this comment from the source in the meantime.

Reply:
Sounds good. Comment removed.